src/streamlink/plugins/pixiv.py

```diff
@@ -56,7 +56,7 @@ def find_videopage(self):
 
         m = self._videopage_re.search(res.text)
         if not m:
-            self.logger.debug("No stream path, stream might be offline or invalid url.")
+            self.logger.debug("No stream path. Offline stream or invalid URL?")
```

Comment:
because I wrote this plugin I am biased,
but this changes doesn't bring any real value.